src/annotator/range-util.js

```diff
@@ -129,3 +129,35 @@ export function selectionFocusRect(selection) {
     return textBoxes[textBoxes.length - 1];
   }
 }
+
+/**
+ * Retrieve a set of items associated with nodes in a given range.
+ *
+ * @template T
```

Comment:
Could you be so kind as to remind me where the documentation is for `@typedef`-enabled jsdoc (or whatever this combination is; I've forgotten!) so I can understand the `@template` tag? Google is not my friend this morning.

Reply:
https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html